src/main/java/com/star_trello/darkside/model/Queue.java

```diff
@@ -0,0 +1,30 @@
+package com.star_trello.darkside.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Builder
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Queue {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    private String title;
+    private String description;
+
+    @ManyToOne
+    private User creator;
```

Comment:
maybe we will hold userId? I don't know, what's the best decision. But it is not necessary to hold whole User, when ID defined User in unique way

Reply:
So, we will need to make 1 more repo request to db: after getting information 'bout queue, u need request user by userId.

Reply:
Or u want to send to front all queues (for example N queues), and after getting that list, u will have to request user by id N times

Reply:
it is not big problem 1 more request. And not N times request from front. Just send array of Id and get array of users. Ok, it is fine too. I hope it will be working